src/components/enterprise-page/data/hooks.js

```diff
@@ -38,8 +45,8 @@ export function useEnterpriseCustomerConfig() {
             branding: {
               logo,
               banner: {
-                backgroundColor: bannerBackgroundColor || defaultBackgroundColor,
-                borderColor: bannerBorderColor || defaultBorderColor,
+                backgroundColor: bannerBackgroundColor,
```

Comment:
What if `brandingConfiguration` exists, including a logo, but `bannerBackgroundColor` and `bannerBorderColor` are null? I believe we'd lose the default colors.

Reply:
hmm okay, I should definitely handle that, and also make the tests capture that, will do